src/scrape_up/Academia/academia_scraper.py

```diff
@@ -0,0 +1,91 @@
+from bs4 import BeautifulSoup
+import requests
+
+class Academia:
+    """
+    ... (class documentation) ...
```

Comment:
Add class documentation, you can check other Classes for reference, how a class is documented in docstring.